app/build.gradle

```diff
@@ -35,15 +36,27 @@ android {
     }
 }
 
+repositories {
+    mavenCentral()
+}
+
 dependencies {
+    implementation 'com.google.maps:google-maps-services:0.15.0'
+    implementation 'org.slf4j:slf4j-simple:1.7.25'
```

Comment:
Whose this for?

Reply:
They said to do so to prevent some errors or something on the google-maps-sevices Github.

Reply:
the first dependency is obviously the google maps services which provides the Java SDK for the API.

Reply:
Then put it as a `runtimeOnly` dependency and make a comment on it. We might switch the implementation type from `simple` later.

Reply:
OK, didn't know it was a thing...

Reply:
I was talking about the `slf4j` not `maps`. `maps` should be `implementation`.

Reply:
OK, obviously I don't know what this actually does...

Reply:
So... This is OK now?

Reply:
yep